devicetypes/smartthings/zigbee-white-color-temperature-bulb.src/zigbee-white-color-temperature-bulb.groovy

```diff
@@ -32,8 +32,8 @@ metadata {
 
 
 		// Generic
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300", outClusters: "0019"
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 0B04", outClusters: "0019"
+		fingerprint profileId: "0104", deviceId: "010C", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300", outClusters: "0019"
```

Comment:
I was thinking that for these generic deviceId specific fingerprints we should add a comment at the end of the fingerprint to identify the deviceId since most people won't know what that means. In this case it would be `Color temperature light`. Do you guys like that? An alternative is no comment or a new join name that says `Generic Color Temperature Light` or something like that.

Reply:
I also think we should remove any clusters that are not used by the DTH. So in this case we only need 0006, 0008 and 0300.

Reply:
I think a comment would be nice, but for the join name I think defaulting to the DTH name is fine since it's basically a generic name.